utils/validation.js

```diff
@@ -0,0 +1,29 @@
+const isValidatedString = (string) => {
+  if (
+    string === null ||
+    string === undefined ||
+    typeof string === "undefined" ||
+    Array.isArray(string) ||
+    typeof string === "object"
+  ) {
+    return false;
+  }
+
+  return true;
+};
+
+const isValidatedNumber = (number) => {
+  if (
+    number === null ||
+    number === undefined ||
+    typeof number === "undefined" ||
+    isNaN(number) ||
```

Comment:
```suggestion
    Number.isNaN(Number(number)) ||
    Number.isNaN(parseInt(number)) ||
```
`Number(value)` function으로 Number type으로 변환 했을 때, NaN이 나오는지 정확히 검사하기 위해 위의 방식 중 하나를 추천드립니다 !

- [참고 DEV blog](https://dev.to/otumianempire/isnan-vs-numberisnan-52gk)
- [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN)

Reply:
엇, 죄송합니다. 착오가 있었던 것 같습니다..! 
문자열 2(`"2"`) 의 엣지케이스에 대해서 생각하지 못하고 말씀드렸네요.

뭔가 실수를 만회하기 위해서 생각해보다가 떠오른 것이, 아래 내용인데요. number 타입이 아닌 자료형을 간단하게 거를 수 있을 것 같습니다.
```js
typeof number !== "number"
```

Reply:
새로운 방법 제안에 수정까지 감사드립니다!
제안해주신 방법대로 적용한다면, `typeof `를 통해 조건을 확인하고 있는 코드 2개를 하나로 통합할 수 있네요!
첫번째로 제안주신 방법은 타입을 정확하게 지켜야하는 DB보단 타입을 구분하지 않아도 되는 로직에서 사용하면 좋을 것 같네요! 
감사합니다!